orangecontrib/text/widgets/owtopicmodeling.py

```diff
@@ -107,6 +107,7 @@ def _run(corpus: Corpus, model: GensimWrapper, state: TaskState):
     def callback(i: float):
         state.set_progress_value(i * 100)
         if state.is_interruption_requested():
+            print("tukaj")
```

Comment:
Probably not something you wanted to include? 😅

Reply:
wow. really? I taught I removed it. Fixed now